tap_github/repository_streams.py

```diff
@@ -1307,3 +1307,55 @@ def post_process(self, row: dict, context: Optional[dict] = None) -> dict:
             ),
         ),
     ).to_dict()
+
+
+class StatsContributorsStream(GitHubStream):
+    """
+    Defines 'StatsContributors' stream. Fetching contributors activity.
+    https://docs.github.com/en/rest/reference/metrics#get-all-contributor-commit-activity
+    """
+
+    name = "stats_contributors"
+    path = "/repos/{org}/{repo}/stats/contributors"
+    primary_keys = ["org", "repo", "user_id"]
+    parent_stream_type = RepositoryStream
+    ignore_parent_replication_key = False
+    state_partitioning_keys = ["repo", "org"]
+    # Note - these queries are expensive and the API might return an HTTP 202 if the response
+    # has not been cached recently. https://docs.github.com/en/rest/reference/metrics#a-word-about-caching
+    tolerated_http_errors = [202]
```

Comment:
@edgarrmondragon the docs say that the request should be retried "a bit later" when we get a 202. Is there some sort of mechanism in the sdk to do this (I'd imagine something like enqueueing retries after N seconds)? I guess it could be handle the same way as a `429 - Too many requests` but that would less than ideal.